Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Climber.java

```diff
@@ -52,7 +52,7 @@ public void attachToBar() {
 	 * stops the climber
 	 */
 	public void stopClimber() {
-		climberMotor.stopMotor();
+		//climberMotor.stopMotor();
```

Comment:
Some commands/other people may call this and expect it to work, maybe change the javadoc?